front/app/modules/commercial/events_widget/admin/EventsWidgetSwitch.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+// Components
+import { WideSectionField } from 'containers/Admin/settings/customize';
+import {
+  Setting,
+  ToggleLabel,
+  StyledToggle,
+  LabelContent,
+  LabelTitle,
+  LabelDescription,
+} from 'containers/Admin/settings/general';
+
+interface Props {
+  checked: boolean;
+  onChange: () => void;
+  title: string;
+  description: string;
+}
+
+export default ({ checked, onChange, title, description }: Props) => (
+  <>
+    <WideSectionField>
+      <Setting>
+        <ToggleLabel>
+          <StyledToggle checked={checked} onChange={onChange} />
+          <LabelContent>
+            <LabelTitle>{title}</LabelTitle>
```

Comment:
Not blocking : I'd have a small preference for moving the title and description to the module too, since I think it doesn't make so much sense to configure them from the main app.